react/features/base/styles/components/styles/PlatformElements.native.js

```diff
@@ -0,0 +1,49 @@
+import { ColorPalette } from './ColorPalette';
+import { BoxModel } from './BoxModel';
+
+import {
+    createStyleSheet
+} from '../../functions';
+
+export const PlatformElements = createStyleSheet({
+
+    /**
+     * Platform specific header button (e.g. back, menu...etc).
+     */
+    headerButton: {
+        alignSelf: 'center',
+        color: ColorPalette.white,
+        fontSize: 26,
+        paddingRight: 22
+    },
+
+    /**
+     * Generic style for a label placed in the header.
+     */
+    headerText: {
+        color: ColorPalette.white,
+        fontSize: 20
+    },
+
+    /**
+     * An empty padded view to place components.
+     */
+    paddedView: {
+        padding: BoxModel.padding
+    },
+
+    /**
+     * The topmost level element of a page.
+     */
+    page: {
+        alignItems: 'stretch',
+        bottom: 0,
+        flex: 1,
+        flexDirection: 'column',
+        left: 0,
+        overflow: 'hidden',
+        position: 'absolute',
```

Comment:
The position absolute is necessary? This cannot be done with some height/width settings or further leveraging flexbox? I see a lot of flexbox being used so it's be nice, if possible, to keep using it for consistency.

Reply:
Yeah, it's a result of a very scientific trial-error procedure of nearly 4 hours :D

Actually this is a side-effect of the recent changes around safe area and all those positioning improvements.